aea/cli/run.py

```diff
@@ -288,7 +288,7 @@ def run(
             addresses=wallet.addresses,
             default_address_key=ctx.agent_config.default_ledger,
         )
-    else:
+    else:  # pragma: no cover
```

Comment:
It is possible to cover this by providing one private key path to the agent only